docs/build-redirects.js

```diff
@@ -7,14 +7,22 @@
  * This script builds FF.com's redirects
  */
 
-const chalk = require("chalk");
-const fs = require("fs-extra");
-const path = require("path");
+import { createRequire } from "node:module";
+const require = createRequire(import.meta.url);
```

Comment:
Since this is a pretty self-contained, just-for-us, package, I feel slightly tempted to use [Import Attributes](https://nodejs.org/docs/latest-v18.x/api/esm.html#import-attributes) even if they're in active development, and the keyword already changed in Node 20. No strong feelings though.

Reply:
Or go with [Tyler's suggestion](https://github.com/microsoft/FluidFramework/pull/20090#discussion_r1522019424).